_config/multiuser.yml

```diff
@@ -2,15 +2,20 @@
 Name: multiuser_editing
 After: framework/routes#coreroutes
 ---
-Director:
+SilverStripe\Control\Director:
   rules:
-    'admin/editing//$Action/$ID': 'MultiUserEditingController'
-LeftAndMain:
+    'multiuser/editing//$Action/$ID': 'Silverstripe\MultiUserEditing\MultiUserEditingController'
+SilverStripe\Admin\LeftAndMain:
   extra_requirements_javascript:
-    - 'multiuser-editing/javascript/multiUserEditing.js'
+    - 'vendor/silverstripe/multiuser-editing/js/multiUserEditing.js'
   extra_requirements_css:
-    - 'multiuser-editing/css/multiuserediting.min.css'
-MultiUserEditingController:
+    - 'vendor/silverstripe/multiuser-editing/css/multiuserediting.min.css'
```

Comment:
`silverstripe/multiuser-editing:css/multiuserediting.min.css`

Reply:
@dhensby  when I do that it comes up with [Emergency] Uncaught InvalidArgumentException: Can't find module 'silverstripe/multiuser-editing'. Is there something I am missing?

Reply:
sorry, it should be `silverstripe/multiuser-editing-alert:...`